heartbeat/_meta/fields.common.yml

```diff
@@ -174,6 +174,22 @@
               type: text
             - name: stack
               type: text
+        - name: screenshot_ref
+          type: group
+          dynamic: false
+          fields:
+            - name: width
+              type: integer
+              description: Width of the full screenshot in pixels.
+            - name: height
+              type: integer
+              description: Height of the full screenshot in pixels
+            - name: blocks
+              type: group
+              fields: 
+                - name: hash
```

Comment:
Need to add `top` and `left` here.

Reply:
Good catch!

Reply:
I don't see the `top` & `left` yet in the new changes.